plank/pkg/server/server.go

```diff
@@ -310,14 +322,61 @@ func (ps *platformServer) SetHttpChannelBridge(bridgeConfig *service.RESTBridgeC
 		Name(fmt.Sprintf("%s-%s", bridgeConfig.Uri, bridgeConfig.Method)).
 		Handler(ps.endpointHandlerMap[endpointHandlerKey])
 	if !atomic.CompareAndSwapInt32(ps.routerConcurrencyProtection, 1, 0) {
-		panic("Concurrency write detected!")
+		panic("Concurrency write on router detected when running ")
 	}
 
 	utils.Log.Infof(
-		"Service channel '%s' is now bridged to a REST endpoint %s (%s)",
+		"[plank] Service channel '%s' is now bridged to a REST endpoint %s (%s)",
 		bridgeConfig.ServiceChannel, bridgeConfig.Uri, bridgeConfig.Method)
 }
 
+// SetHttpPathPrefixChannelBridge establishes a conduit between the the transport service channel and a path prefix
+// every request on this prefix will be sent through to the target service, all methods, all sub paths, lock, stock and barrel.
+func (ps *platformServer) SetHttpPathPrefixChannelBridge(bridgeConfig *service.RESTBridgeConfig) {
```

Comment:
this is largely identical to `SetHttpChannelBridge()` can we add an additional boolean flag to that method instead to allow the wildcard methods mode?

Reply:
Thats a breaking change, any signature changes are always a last resort for me. There are some differences in here however, all methods are permitted and the router uses `PathPrefix` instead of `Path`. Sometimes, I really miss overloading and once day hope it makes it into golang. I made this choice, to preserve backwards compatibility.